pkg/ansible/runner/runner.go

```diff
@@ -250,6 +250,8 @@ func (r *runner) Run(ident string, u *unstructured.Unstructured, kubeconfig stri
 			dc = r.cmdFunc(ident, inputDir.Path)
 		}
 		dc.Env = append(dc.Env, fmt.Sprintf("K8S_AUTH_KUBECONFIG=%s", kubeconfig), fmt.Sprintf("KUBECONFIG=%s", kubeconfig))
+		// Append current environment since setting dc.Env to anything other than nil overwrites current env
+		dc.Env = append(dc.Env, os.Environ()...)
```

Comment:
I think this should go before line 252, where the other `append` happens. The docs for `Cmd.Env` say "If Env contains duplicate environment keys, only the last value in the slice for each duplicate key is used." In case there happens to be a `KUBECONFIG` or similar key already in the current process's environment, we want the ones being set above to win.

Reply:
Ah great catch. Thanks!